rscripts/metafeature_nucleotides.R

```diff
@@ -0,0 +1,628 @@
+# Uses the fasta file, gff and h5 file to produce a metafeature window averageing over positions of interest.
+# The positions to be averaged across should be provided by the user as a TSV file with the headings Gene and Pos
+# The TSV will provide info on the gene and nt position relative to the start codon for each of the features being averaged across
+
+# For testing, using a tsv file containing the following lines:
+
+# Gene	Pos
+# MIKE	5
+# MAT	7
+# MAT	5
+# MAT	6
+
+
+print('Starting process')
+
+source(here::here("rscripts", "read_count_functions.R"))
+source(here::here("rscripts", "stats_figs_block_functions.R"))
+
+library(Biostrings)
+library(rtracklayer)
+library(stringr)
+library(GenomicRanges)
+library(parallel)
+library(rhdf5)
+library(dplyr)
+
+suppressMessages(library(optparse))
+
+option_list <- list(make_option(c('-i', '--input'),type = "character", help='Path input to h5 file'),
+                    make_option(c('-d', '--dataset'),type = "character", help='Name of the dataset being studied'),
+                    make_option(c('-g', '--gff'),type = "character", help='Path to the GFF3 file of the organism being studied'),
```

Comment:
Should we be calling it an organism or a species

Reply:
I have changed it to species